### PR TITLE
CPU Stress Experiment

### PR DESCRIPTION
**Pull request by neelanjan00**
- Adds CPU Stress Experiment Charts
- Merges `master` branch with `m-agent` branch

### CODE REVIEW

**Review by uditgaurav — APPROVED** [review]

File: charts/os/cpu-stress/engine.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: cpu-stress--chaos
```
Comment (uditgaurav) [inline charts/os/cpu-stress/engine.yaml]:
```suggestion
  name: cpu-stress-chaos
```

File: charts/os/cpu-stress/engine.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+spec:
+  engineState: 'active'
+  chaosServiceAccount: cpu-stress-sa
+  jobCleanUpPolicy: 'delete'
```
Comment (uditgaurav) [inline charts/os/cpu-stress/engine.yaml]:
```suggestion
  jobCleanUpPolicy: 'retain'
```

We can remove this field - by default it is retain

File: charts/os/cpu-stress/experiment.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        verbs: ["get","list","watch"]  
+      # for creating and managing to execute commands inside target container
+      - apiGroups: ["batch"]
+        resources: ["jobs"]
+        verbs: ["create","list","get","delete","deletecollection"]
+      # for creation, status polling and deletion of litmus chaos resources used within a chaos workflow
+      - apiGroups: ["litmuschaos.io"]
+        resources: ["chaosengines","chaosexperiments","chaosresults"]
+        verbs: ["create","list","get","patch","update","delete"]
+    image: "litmuschaos/go-runner:m-agent"
```
Comment (uditgaurav) [inline charts/os/cpu-stress/experiment.yaml]:
```suggestion
    image: "litmuschaos/go-runner:2.10.0-Beta1"
```

File: charts/os/cpu-stress/experiment.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    env:
+
+    - name: TOTAL_CHAOS_DURATION
+      value: '30' 
+
+    - name: CHAOS_INTERVAL
+      value: '30'
+
+    - name: LIB
+      value: ''
```
Comment (uditgaurav) [inline charts/os/cpu-stress/experiment.yaml]:
```suggestion
      value: 'litmus'
```

**Review by avaakash — APPROVED** [review]

**Pull request merged**